pkg/cmd/diagnostics/diagnostics_create.go

```diff
@@ -10,14 +10,18 @@ import (
 )
 
 func newDiagnosticsCreateCommand(ctx api.Context) *cobra.Command {
+	var mastersOnly bool
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create a diagnostics bundle",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := pluginutil.HTTPClient("")
 			client := diagnostics.NewClient(c)
-
-			id, err := client.Create()
+			opts := diagnostics.Options{
+				Masters: true,
+				Agents:  !mastersOnly,
```

Comment:
Let's think about a possible next step of having `--agents` as well. How would that work? I'd suggest this:

`--masters`: only master nodes
`--agents`: only agent nodes
`--masters --agents`: masters and agents
none: same as `--masters --agents`

Then the code we'd have here would be:

```
opts := diagnostics.Options{
    Masters: mastersOnly || !mastersOnly && !agentsOnly
    Agents: agentsOnly || !mastersOnly && !agentsOnly
}
```
In that case naming the flags `masters` and `agents` instead of `...Only` probably makes more sense.

wdyt?

Reply:
Do we need `--agents` flag? The difference between `--agents` and `--masters --agents` is only in 3/5 nodes

Reply:
https://jira.mesosphere.com/browse/DCOS_OSS-5547 actually says we can add `--agents` later which makes me think we should at least make it easy to add it later. AFAIK leaving out masters may drastically reduce bundle size, doesn't it?

Reply:
Thats true masters are the slowest and heaviest nodes in terms of bundle generation